webpack/scenes/ContentViews/components/EnvironmentPaths/EnvironmentPaths.js

```diff
@@ -41,9 +43,11 @@ const EnvironmentPaths = ({ userCheckedItems, setUserCheckedItems }) => {
             <FormGroup key={`fg-${count}`} isInline fieldId="environment-checkbox-group">
               {environments.map(env =>
                 (<Checkbox
-                  isChecked={env.library ||
-                  userCheckedItems.filter(item => item.id === env.id).length}
-                  isDisabled={env.library}
+                  isChecked={(publishing && env.library) ||
```

Comment:
Can we create a variablefor these: 
 promotedEnvironments.filter(item => item.id === env.id).length

It's repeated twice and for outside readability, if it was named something like const hasPromotedEnvironment?
Thoughts?

Reply:
+1 it would communicate the intent of the code much more clearly

Reply:
Updated to use method for these checks..